### ISSUE TITLE
Unexpected comma in middle of array - no-sparse-arrays

### ISSUE THREAD

**Issue by moso** [nuxt/create-nuxt-app#181]
Hey, I stumbled upon this while creating a new Nuxt 2.4.2 app.

### Steps to reproduce
`yarn create nuxt-app`

Go through the install, under **features** select all, deselect one (mine was `a` for all, deselect `Axios`), continue.

My entire process:

- Project name: project-name
- Project description: Project description
- Use a custom server framework: none
- Choose features to install: PWA, Linter/Formatter, Prettier
- Use a custom UI framework: none
- Use a custom test framework: none
- Choose rendering mode: universal
- Author name: name
- Choose a package manager: yarn

### What is expected?
The `create-nuxt-app` to finish without errors

### What actually happened?
`error  Unexpected comma in middle of array  no-sparse-arrays` in `nuxt.config.js`

`cat nuxt.config.js` reveals this extra comma in the modules array:

```javascript
  /*
  ** Nuxt.js modules
  */
  modules: [, '@nuxtjs/pwa'],
```

I assume it's left from when I deselected Axios.

**Comment by okaybroda** [comment]
This happened to me too. Is it safe to continue?

**Comment by manniL** [comment]
For now, feel free to delete the comma and go ahead :relaxed: 

Thanks for the report

**Comment by clarkdo** [comment]
Fixed by #172
